README.md

```diff
@@ -1,62 +1,83 @@
 
-## Goals
-* Make it easy for users to share information publicly, while preserving privacy when desired
-* Improve the onboarding experience for decentralized applications by making it easy for developers to get information about Ethereum users
-* Improve Ethereum usability with distributed database infrastructure that works in production today
-* Provide a decentralized system, but pragmatically utilize centralized components to facilitate the transition
+[`3box-graphql`](https://github.com/3box/3box-js-graphql) is a GraphQL endpoint that allows developers to write more efficient 3Box.js getProfile() queries. Now developers can ask our API for specific common user profile fields instead of needing to return the entire profile. For example, this is useful when querying for name and image for hundreds or thousands of profiles at once.
 
-## 3Box DB
-3Box DB consists of two components: [`3box-js`](https://www.github.com/3box/3box-js) which is a client library, and [`3box-address-server`](https://www.github.com/3box/3box-address-server) which is a server. Together they are used to keep track of user data stored in [`orbit-db`](https://github.com/orbitdb/orbit-db) instances. Learn more about the [3Box architecture and how we do access control with encryption](./ARCHITECTURE.md).
+## 3Box Data Network
+The 3Box user data network consists of a few core components: the [pinning server](https://www.github.com/3box/3box-pinning-server), the [address server](https://www.github.com/3box/3box-address-server), and a [caching service](https://www.github.com/3box/3box-caching-service). Together these components are used to keep track of user data stored in [`orbit-db`](https://github.com/orbitdb/orbit-db) instances on IPFS. 
 
-### 3Box JS
-[`3box-js`](https://www.github.com/3box/3box-js) is the client side library used to read, write, and delete public and private data associated with the user. This library is used to get profile information about an address.
+We assume that end-users have a web3-compatible browser or wallet (such as MetaMask or Status) which supports personal_sign. This is how users authenticate dapps to their 3Box. Learn more about the [3Box architecture and how we do access control with encryption](./ARCHITECTURE.md).
 
-We assume that end users have a web3-compatible browser or wallet (such as MetaMask or Status) which supports personal_sign.
+### 3Box Pinning Server
+[`3box-pinning-server`](https://www.github.com/3box/3box-pinning-server) is a service operated by 3Box that pins 3Box user data on the IPFS network to ensure availability. In the future this service will become more decentralized with a network of pinning nodes, allowing others to run their own node and host their own data if desired.
```

Comment:
We should add something like: "Right now others can run their own instances of the pinning-server to duplicate the data in the 3box user data network"